panther_battery/README.md

```diff
@@ -1,64 +1,37 @@
-[//]: # (ROS_API_PACKAGE_START)
-[//]: # (ROS_API_PACKAGE_NAME_START)
-
 # panther_battery
 
-[//]: # (ROS_API_PACKAGE_NAME_END)
-[//]: # (ROS_API_PACKAGE_DESCRIPTION_START)
-
-Package containing nodes monitoring and publishing the internal battery state of the Husarion Panther robot.
-
-[//]: # (ROS_API_PACKAGE_DESCRIPTION_END)
+The package containing nodes monitoring and publishing the internal battery state of the Husarion Panther robot.
 
-## ROS Nodes
+## Launch Files
 
-[//]: # (ROS_API_NODE_START)
-[//]: # (ROS_API_NODE_COMPATIBLE_1_0)
-[//]: # (ROS_API_NODE_COMPATIBLE_1_2)
-[//]: # (ROS_API_NODE_NAME_START)
+This package contains:
 
-### battery_driver
+- `battery.launch.py`: Responsible for activating battery node, which dealing with reading and publishing battery data.
 
-[//]: # (ROS_API_NODE_NAME_END)
-[//]: # (ROS_API_NODE_DESCRIPTION_START)
+### battery_node
```

Comment:
It's bettery_driver now 🤔 
Do we refer to file or node names that appear in the stack? I think this question applies to several similar cases in this PR.

Reply:
Imo should we refer to default executable name no renamed node, because this README is package description not whole system.